amd64/root/start.sh

```diff
@@ -33,16 +33,16 @@ echo "[marthoc/deconz] Checking device group ID"
 if [ "$DECONZ_DEVICE" != 0 ]; then
   DEVICE=$DECONZ_DEVICE
 else
- if [ -f /dev/ttyUSB0 ]; then
+ if [ -e /dev/ttyUSB0 ]; then
```

Comment:
I just recognized that the `DEVICE` set by these statements is not added to the options `DECONZ_OPTS ` at the end of this script:
```sh
if [ "$DECONZ_DEVICE" != 0 ]; then
  DECONZ_OPTS="$DECONZ_OPTS --dev=$DECONZ_DEVICE"
fi
```

Reply:
It's indeed the case, as it would change the current functionality. 

The only reason to see which device is available is to check which group it has and use the groupid.